app/views/events/edit.html.erb

```diff
@@ -11,3 +13,17 @@
     <% end %>
   </div>
 </div>
+
+
+
+<script>
+  document.addEventListener("DOMContentLoaded", function() {
+    const triggerButton = document.querySelector(".trigger-copy");
+    const realCopyButton = document.getElementById("real-copy-button");
+    
+    triggerButton.addEventListener("click", function(e) {
+      e.preventDefault(); 
+      realCopyButton.click(); 
+    });
```

Comment:
if you do not check for presence or do not use the variables more than once, we could get rid of them i guess. what do you think would be the benefit of keeping them instead of the following:

(we can also use jQuery to reduce code)

```
$('#trigger-copy').on('click', function(e) {
  e.preventDefault();
  $('#real-copy').click();
});
```

Reply:
good point

Reply:
fixed it a5322f35e7a3650f3ced2025df8f64a20c611678